tests/test_resolutions.py

```diff
@@ -123,8 +123,7 @@ def test_can_be_instantiated(self) -> None:
 
     def test_the_test(self) -> None:
         cim = ContainsItemMatching(r"([Ss]pam ?)+").resolve()
-
-        assert cim.matches(["Spam", "Eggs", "Spam and eggs"])
+        assert cim.matches(["Spam", "Eggs", "Spam and eggs"], )
```

Comment:
Can you explain this change?

Reply:
erm.. no I can't. That's gotta be black maybe?  or my IDE did something funky.  I'll fix it.